src/libcmd/markdown.cc

```diff
@@ -51,9 +53,21 @@ std::string renderMarkdownToTerminal(std::string_view markdown)
         throw Error("allocation error while rendering Markdown");
 
     return filterANSIEscapes(std::string(buf->data, buf->size), !isTTY());
+}
+
+std::string renderMarkdownToTerminal(std::string_view markdown)
+{
+    if (auto e = getEnv("_NIX_TEST_RAW_MARKDOWN"); e && *e == "0")
+        return doRenderMarkdownToTerminal(markdown);
+    else
+        return std::string(markdown);
```

Comment:
I am confused on the condition, seems reversed or something? I would think if no env var is defined, we would want the markdown to be rendered?

Reply:
raw != rendered  (perhaps?)

Reply:
Yeah what the
I was too focused on getting the damn thing to pass.